src/osd/OSD.cc

```diff
@@ -2331,6 +2331,8 @@ int OSD::init()
       MPGStats *m = new MPGStats(monc->get_fsid(), osdmap->get_epoch(), had_for);
       m->osd_stat = cur_stat;
 
+      min_last_epoch_clean = osdmap->get_epoch();
+      min_last_epoch_clean_pgs.clear();
```

Comment:
the are defined as member variables of `this`. and we need to reset them every time.